applications/nrf_desktop/doc/config_channel.rst

```diff
@@ -223,12 +223,12 @@ Recipient discovery operations
 Response
 --------
 
-In case of response, the value of the Status field indicates the state of the earlier request.
+In the case of a response, the value of the Status field indicates the state of the earlier request.
 The following values are possible:
 
-* ``CONFIG_STATUS_PENDING`` - The operation is not yet completed.
+* ``CONFIG_STATUS_PENDING`` - The operation is not yet complete.
```

Comment:
I would use "has not completed" here since the next sentence also has "the operation completes" instead of "becomes complete."